x86_energy_plugin.cpp

```diff
@@ -637,7 +581,7 @@ class x86_energy_plugin
 private:
     const std::string prefix_ = "x86_energy/";
     scorep::plugin::metric_property add_metric_property(const std::string& name, 
-            int sensor, int node, std::string quantity)
+            int sensor, int node, std::string& quantity)
```

Comment:
`std::string& quantity` should be `const std::string&`. Excuse me, if I missed the const.